### DISCUSSION TITLE
Trascr_019: indicazioni "autorità mittente - "atto da trascrivere"

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1040] [Supporto servizi cooperativi]
Il caso d'uso
_Trascr_019 - Trascrizione di nascita: Trascrizione del provvedimento straniero di adozione di minore riconosciuto valido dal Tribunale dei minori convenzione Aja 29.5.1993_
prevede la presenza della sezione "obbligatoria" della sezione "Atto da trascrivere":

![image](https://github.com/user-attachments/assets/c2ace92f-99e0-43d1-a28c-8498d9b0c6b7)

E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al provvedimento estero di adozione?

Inoltre, si prevede la presenza della sezione "obbligatoria" della sezione "Autorità mittente":

![image](https://github.com/user-attachments/assets/6741ea5e-1741-4049-9e83-728381240637)

E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al Tribunale dei minori che disponga la trascrizione del provvedimento di adozione all'interno dei registri di stato civile?

**Nota**: l'iter completo di gestione del processo di adozione laddove il paese straniero aderisca alla convenzione Aja dovrebbe essere quello riassunto nella pagina ANUSCA https://www.anusca.it/flex/cm/pages/ServeBLOB.php/L/IT/IDPagina/3651

**Comment by acolosimosogei** [comment]
Salve, 
riporto di seguito le domande poste nella discussion seguite dalle relative risposte:

1) E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al provvedimento estero di adozione?

   E' corretto


2) E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al Tribunale dei minori che disponga la trascrizione del provvedimento di adozione all'interno dei registri di stato civile?

   Corretto


Chiudiamo pertanto la discussion: in caso di ulteriori richieste si prega di aprirne un'altra dedicata.
Grazie!
